docs/source/release/v6.13.0/Direct_Geometry/CrystalField/Bugfixes/39039.rst

```diff
@@ -0,0 +1 @@
+- Removed unnecessary fix for IB21 for C1 and Ci symmetries in the Crystal Field peak base function.
```

Comment:
Can we include a ref to the docs for the changed fit function? https://docs.mantidproject.org/nightly/fitting/fitfunctions/CrystalFieldPeaks.html

Reply:
I agree that a reference to a document might be useful. How about the parameter tables from the concept page (https://docs.mantidproject.org/nightly/concepts/CrystalField.html#appendix-b-tables)?